### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by buaa-luzhi** [kilsen512/R3DCNN-tensorflow#8]
Firstly, I am so glad that you can help me again.
For the past two years, I have also been doing isolated gesture recognition. I've also faced the problems with no gesture class. Later, we collected a lot of video files that did not contain hands to represent the no gesture class. Therefore, the total output of the softmax classifier is equal to the 'predefined class' + 'no gesture class'.
I'd like to know what the predefined and non gesture video files of training network look like in your project. I am eager for you to send two examples of video files to my email, by1603117@buaa.edu.cn
Thanks so much for your help.
Besides, for https://github.com/breadbread1984/R3DCNN, I have trained many times. I don't know why every time I test the local video using the pre-training model the output is random.
![1](https://user-images.githubusercontent.com/28650172/71073031-95423800-21ba-11ea-8551-ce7b964d6285.png)

The picture above shows the test results of one experiment. In addition, the code is too encapsulated to be easily modified.
Therefore, thanks for your sharing.

Besides, in addition, is the 'no gesture' class required by the project or set by using CTC. As far as I know, the concept of 'no gesture' class is introduced in the original text for the purpose of using CTC, and the actual training data is not set with 'no gesture' class alone.
Thanks again. I'd really like to ask you more questions. Please forgive me.
